alerts/nginx/README.md

```diff
@@ -0,0 +1,42 @@
+# Nginx Alerts for Ops Agent
+
+### Notification Channels
+For all alerts, a notification channel needs to be set up or the alert will fire silently.
+
+### User Labels
+User labels can be used for these policies by modifying the userLabels fields of the policies. i.e.
+
+```json
+{ 
+  "userLabels": {
+    "datacenter": "central"
+  }
+}
+```
+
+## High connections dropped alert
```

Comment:
Could we move the description of the alert policies above the implementation details (notifications and user labels)? Thanks!

Reply:
fixed!